akka-projection-persistence-query-provider/src/main/scala/akka/projection/provider/AkkaPersistenceQuerySourceProvider.scala

```diff
@@ -0,0 +1,50 @@
+package akka.projection.provider
```

Comment:
Not so sure about this package name. 

Alternatively, `akka.projection.query`? 

I'm afraid people could think it has something to do with querying the read-side.